core/io/file_access_encrypted.cpp

```diff
@@ -49,6 +49,15 @@ Error FileAccessEncrypted::open_and_parse(Ref<FileAccess> p_base, const Vector<u
 		writing = true;
 		file = p_base;
 		key = p_key;
+		if (p_iv.is_empty()) {
+			iv.resize(16);
+			for (int i = 0; i < 16; i++) {
+				iv.write[i] = Math::rand() % 256;
+			}
+		} else {
```

Comment:
I don't know the full context here, but does the init-vector not need to be _securely_ random? As repeated invocations of `Math::rand()` probably have rather low entropy...

Reply:
I do not think it matters in this case, but changed it to `CryptoCore::RandomGenerator` just in case.